crates/edr_evm/src/blockchain/forked.rs

```diff
@@ -55,7 +55,7 @@ pub enum CreationError {
     },
 }
 
-/// Error type for [`RemoteBlockchain`].
+/// Error type for `RemoteBlockchain`.
```

Comment:
To properly fix this, I had to make the `remote` module public, which in turn meant documenting it and its public things... so I just removed the link here.

Reply:
See https://github.com/NomicFoundation/hardhat/pull/5074#discussion_r1551178330